lms/djangoapps/discussion/views.py

```diff
@@ -447,6 +453,7 @@ def _create_discussion_board_context(request, course_key, discussion_id=None, th
         'upgrade_link': check_and_get_upgrade_link(request, user, course.id),
         'upgrade_price': get_cosmetic_verified_display_price(course),
         # ENDTODO
+        'discussion_default_topic_id': _get_discussion_default_topic_id(course),
```

Comment:
Could you add a comment that None is an acceptable value here, in which case the front-end code will look for a topic that starts with "General" to use as the default? It took me some time to look through the code to understand what would happen if _get_discussion_default_topic_id did not return an entry ID.